core/src/main/java/org/bitcoinj/coinjoin/utils/ProTxToOutpoint.java

```diff
@@ -0,0 +1,477 @@
+/*
+ * Copyright (c) 2022 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bitcoinj.coinjoin.utils;
+
+import org.bitcoinj.core.Context;
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.core.Sha256Hash;
+import org.bitcoinj.core.TransactionOutPoint;
+
+import java.util.HashMap;
+
+public class ProTxToOutpoint {
+
```

Comment:
This class helps us get around the fact that in Core 18, the dsq message contains a `masternodeOutpoint` instead of a proTxHash to identity a masternode. 

So for now we need these values hard coded.  

This solution works fine on a devnet (small network), it will work OK temporarily on a testnet.  It is not a good solution on mainnet.  Overtime these hardcoded values become stale and will require updating.

In Core 19, the issue that this solves should be resolved and we won't need this class, a class in `tools` or some other resources in this PR.